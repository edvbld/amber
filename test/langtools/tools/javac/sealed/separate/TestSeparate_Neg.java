/*
 * @test /nodynamiccopyright/
 * @summary smoke test for sealed and separate compilation
 * @compile/fail/ref=TestSeparate_Neg.out -XDrawDiagnostics TestSeparate_Neg.java SealedClasses.java
 * @clean *
 * @compile SealedClasses.java
 * @compile/fail/ref=TestSeparate_Neg.out -XDrawDiagnostics TestSeparate_Neg.java
 */

public class TestSeparate_Neg {
    static class C_SCN extends SealedClasses.SC { }
    static class C_SACN extends SealedClasses.SAC { }
    static class C_SIN implements SealedClasses.SI { }
    static interface I_SIN extends SealedClasses.SI { }
}
