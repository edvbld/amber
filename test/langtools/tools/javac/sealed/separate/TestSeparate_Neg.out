TestSeparate_Neg.java:8:45: compiler.err.cant.inherit.from.sealed: SealedClasses.SC
TestSeparate_Neg.java:9:46: compiler.err.cant.inherit.from.sealed: SealedClasses.SAC
TestSeparate_Neg.java:10:48: compiler.err.cant.inherit.from.sealed: SealedClasses.SI
TestSeparate_Neg.java:11:49: compiler.err.cant.inherit.from.sealed: SealedClasses.SI
4 errors
